test/hash.test.js

```diff
@@ -73,6 +73,10 @@ describe("Currency", function () {
     const xrp = new Currency(Buffer.alloc(20));
     expect(xrp.iso()).toBe("XRP");
   });
+  test("Can handle non-standard currency codes", () => {
+    const currency = "015841551A748AD2C1F76FF6ECB0CCCD00000000";
```

Comment:
let's make this a constant at the top of the test file

Reply:
Should I do that with all the tests then? It seems weird to just do it for a few